pages/collection.html

```diff
@@ -147,6 +147,7 @@ <h1 id="title-header">The Regis Altare Birthday Project 2023</h1>
             <option value="all">All</option>
           </select>
         </label>
+        <input type="search" name="search" id="search-bar" placeholder="Search by name/artist/writer..." aria-label="Search cards by name, artist or writer" size="23" maxlength="32">
```

Comment:
Any reasoning behind the max length choice?

Reply:
Letting the users type input without character limit could lead to some performance issues; I'm not certain on that, but regardless 32 characters should be more than enough to narrow down the search, so I placed the limit just in case.